test/deal-investor/get-deal-investors-by-deal-identifier.api-test.ts

```diff
@@ -0,0 +1,112 @@
+import { UkefId } from '@ukef/helpers';
+import { withAcbsAuthenticationApiTests } from '@ukef-test/common-tests/acbs-authentication-api-tests';
+import { Api } from '@ukef-test/support/api';
+import { ENVIRONMENT_VARIABLES } from '@ukef-test/support/environment-variables';
+import { RandomValueGenerator } from '@ukef-test/support/generator/random-value-generator';
+import nock from 'nock';
+
+import { DealInvestorGenerator } from '../support/generator/deal-investor-generator';
+
+describe('GET /deals/{dealIdentifier}/deal-investors', () => {
```

Comment:
Sorry for not raising this yesterday, only just noticed - shouldn't the endpoint be '.../investors', not '.../deal-investors'?

Reply:
Changed.